src/core/annotation.js

```diff
@@ -633,16 +626,24 @@ class MarkupAnnotation extends Annotation {
     this.data.title = stringToPDFString(dict.get('T') || '');
 
     if (dict.has('IRT')) {
-      this.data.inReplyTo = dict.getRaw('IRT').toString();
-      this.data.replyType =
-      dict.has('RT') ? dict.get('RT').name : AnnotationReplyType.REPLY;
+      const rawIRT = dict.getRaw('IRT');
```

Comment:
What type is this supposed to be?

If `Ref`, then you need to check `isRef(...)` below; similar for other types of course.

Reply:
Honestly, I'm not sure … it's the same thing as parent in popup, so I used same approach as the one that is already in master ([https://github.com/mozilla/pdf.js/blob/master/src/core/annotation.js#L989](https://github.com/mozilla/pdf.js/blob/master/src/core/annotation.js#L989)). But it is probably Ref … so I do it like this? 
isRef(dict.get("IRT")) ? dict.getRaw("IRT").toString() : null

Reply:
> isRef(dict.get("IRT")) ? dict.getRaw("IRT").toString() : null

That won't work at all since `dict.get` dereferences `Ref` objects, whereas `dict.getRaw` doesn't as its name suggests.
```javascript
const rawIRT = dict.getRaw('IRT');
this.data.inReplyTo = isRef(rawIRT) ? rawIRT.toString() : null;
```

Reply:
I'm sorry ... I'm really tired ... after I looked into get method you showed me, it hit me :). Thanks.